docs/pages/guides/ReduxForm.example.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import Code from '../../_shared/Code';
+import { connect } from 'react-redux';
+import { Grid } from '@material-ui/core';
+import { KeyboardDatePicker } from '@material-ui/pickers';
+import { reduxForm, Field, formValueSelector } from 'redux-form';
+
+const DateField = props => {
+  const {
+    meta: { submitting, error, touched },
+    input: { onBlur, value, ...inputProps },
+    ...others
+  } = props;
+
+  return (
+    <KeyboardDatePicker
+      {...inputProps}
+      {...others}
+      format="dd/MM/yyyy"
+      value={value ? new Date(value) : null}
+      disabled={submitting}
+      onBlur={() => onBlur(value ? new Date(value).toISOString() : null)}
+      error={error && touched}
+      onChange={date => Date.parse(date) && inputProps.onChange(date.toISOString())}
+    />
+  );
+};
+
+class ReduxFormExample extends React.PureComponent {
+  render() {
+    return (
+      <form>
+        <Grid container>
+          <Grid item container justify="center" xs={12}>
+            <Field name="date" component={DateField} />
+          </Grid>
+          <Grid item xs={12} sm={12} style={{ margin: '24px' }}>
+            <Code
+              children={JSON.stringify(
+                {
+                  values: {
```

Comment:
I think here we can pass some values from props. Actually redux-form's values and errors. 
So it will be more clear for users

Reply:
most of redux form props are on Field level, I can't get in a simple way the error text outside the Field component... so I add a valid props from Redux-Form, if the value is "true" no errors were find on the form